File: 3358.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');
let g = parseInt(lines.shift()); let soma = 0; let n = ''; let h = '';
for(var i = 0; i < g; i++){
    soma = 0;
    n = lines.shift();
    h = n.toLowerCase();
    for(var j = 0; j < h.length; j++){
        if(soma >= 3){
            break;
        }
        if(h[j] !== 'a' && h[j] !== 'e' && h[j] !== 'i' && h[j] !== 'o' && h[j] !== 'u'){
            soma += 1;
        }
        else{
            soma = 0;
        }
    }
    if(soma >= 3){
        console.log(`${n} nao eh facil`);
    }
    else{
        console.log(`${n} eh facil`);
    }
}

File: 3358.py
for i in range(0, int(input())):
    soma = 0
    n = input().lower()
    for j in range(0, len(n)):
        if soma >= 3:
            break
        if n[j] != 'a' and n[j] != 'e' and n[j] != 'i' and n[j] != 'o' and n[j] != 'u':
            soma += 1
        else:
            soma = 0
    print(f'{n.capitalize()} nao eh facil') if soma >= 3 else print(f'{n.capitalize()} eh facil')
